ntbs-service/wwwroot/css/table.scss

```diff
@@ -10,7 +10,23 @@
 // NHS UK tables have changed their internal padding.
 // See https://nhsuk.github.io/nhsuk-frontend/components/tables/tables-panel.html
 // Overall it's an improvement, but the lack of LHS space looks odd in the first column - this hack alleviates that.
-.nhsuk-table__header:first-child,
-.nhsuk-table__cell:first-child {
-  padding-left: 8px;
-}
\ No newline at end of file
+.nhsuk-table__header:first-child, .nhsuk-table__cell:first-child {
+    padding-left: 8px;
+}
+.nhsuk-table__header:last-child, .nhsuk-table__cell:last-child {
+    padding-right: 8px !important;
```

Comment:
Why do we need to use `important` here? What rule are we competing against? Is there no other way to increase our specificity?

If you do need to use `important` then at a minimum I think you should add a comment explaining why.

Reply:
I've had another look at it and found a way of increasing the specificity of the rules. We were competing against a style from the NHS library, but I've found a way of sorting it without using `!important`